python/ray/autoscaler/v2/autoscaler.py

```diff
@@ -90,7 +93,13 @@ def _init_cloud_provider(
 
         """
         provider_config = config.get_provider_config()
-        if config.provider == Provider.READ_ONLY:
+        if provider_config["type"] == "kuberay":
+            provider_config["head_node_type_name"] = config.get_head_node_type()
+            self._cloud_provider = KubeRayProvider(
+                config.get_config("cluster_name"),
+                provider_config,
+            )
+        elif config.provider == Provider.READ_ONLY:
```

Comment:
What does "READ_ONLY" mean?

Reply:
It's a static cluster that's needed for local cluster: https://github.com/ray-project/ray/pull/43355/files#diff-505e5015f51dfa9726617e4d7f39d53f360e4893ee1ac1052615cef8f265565bR1

It's also in V1.